editor/src/components/navigator/navigator-item/component-picker.tsx

```diff
@@ -147,6 +156,37 @@ export const ComponentPicker = React.memo((props: ComponentPickerProps) => {
     [flatComponentsToShow, highlightedComponentKey, onItemClick],
   )
 
+  const fakeCategorizedComponents = [
+    {
+      label: FILTER_CATEGORIES[0],
+      items: flatComponentsToShow.slice(0, 1),
+    },
+    {
+      label: FILTER_CATEGORIES[1],
+      items: flatComponentsToShow.slice(1, 5),
+    },
+    {
+      label: FILTER_CATEGORIES[2],
+      items: flatComponentsToShow.slice(5, 7),
+    },
+    {
+      label: FILTER_CATEGORIES[3],
+      items: flatComponentsToShow.slice(7, 8),
+    },
+    {
+      label: FILTER_CATEGORIES[4],
+      items: flatComponentsToShow.slice(8, 9),
+    },
+    {
+      label: FILTER_CATEGORIES[5],
+      items: flatComponentsToShow.slice(9, 10),
+    },
+    {
+      label: FILTER_CATEGORIES[6],
+      items: flatComponentsToShow.slice(10),
+    },
+  ]
```

Comment:
Randomly assigning components to categories, will need to be replaced by API.

This static assignment can still be active if we just hide the UI for the tabs and shouldn't provide any extra functionality.